package/functions/funcs/removeNewLines.js

```diff
@@ -0,0 +1,14 @@
+module.exports = async (d) => {
+  const code = d.command.code;
+
+  const r = code.split("$removeNewLines").length - 1;
```

Comment:
@DevSpen  hmm you could use unpack() function directly Instead of using the old method so 
it would have been reduced to 

```js
const inside = d.unpack()

Reply:
Not sure what you mean, can you explain further? 😅

Reply:
It's not really a problem, as long as it works

Reply:
well fair enough

Reply:
i just meant that 
```js
const r = code.split(....) and 
const inside = ...
``` 
in ur code can be simplified to 
const inside = d.unpack()